transcom/mymove
# MB-2617 TIO can see service item card with details - Payment request

## Description

> Given that I am a a TIO
 When I view the "Payment Requests" tab of the Office interface
 Then I see the first Service Item's info in a panel
  And I can click "Back to payment requests" to return to the queue

## Reviewer Notes

- We're using test data for now
- This will use the payment requests skeleton we have
- Design: Made a couple container updates that affected other components. Please let me know if this is an issue.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

To test:

1. Log in to office app, TIO role
2. Navigate to [http://officelocal:3000/moves/1/payment-requests](http://officelocal:3000/moves/1/payment-requests)
3. Click on any payment request id
4. Should land on `http://officelocal:3000/moves/1/payment-requests/[paymentRequestId]`

## Code Review Verification Steps


## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2617) for this change

## Screenshots

![image](https://user-images.githubusercontent.com/1522549/87605670-12c91a80-c6ae-11ea-91ea-f39f33f3e58b.png)
